install/Knative-custom-install.md

```diff
@@ -60,26 +57,28 @@ If you are just getting started with Knative, you should choose automatic
 sidecar injection and enable the Istio service mesh.
 
 Due to current dependencies, some installable Knative options require the Istio
-service mesh. If you install any of the following options, you must install the
-`istio.yaml` package so that automatic sidecar injection is enabled:
+service mesh. If you install any of the following options, you must install
+`istio.yaml` so that automatic sidecar injection is enabled:
 
 - [Knative Eventing](https://github.com/knative/eventing)
 - [Knative Eventing Sources](https://github.com/knative/eventing-sources)
 - [Observability plugins](../serving/installing-logging-metrics-traces.md)
 
-#### Istio installation packages
+#### Istio installation options
 
-| Istio Package Filename | Description                                                            |
-| ---------------------- | ---------------------------------------------------------------------- |
-| [`istio-crds.yaml`][a] | Initially create CRDs before installing Istio.                         |
-| [`istio.yaml`][b]      | Install Istio with service mesh enabled (automatic sidecar injection). |
-| [`istio-lean.yaml`][c] | Install Istio and disable the service mesh by default.                 |
+| Istio Install Filename     | Description                                                            |
+| -------------------------- | ---------------------------------------------------------------------- |
+| **[`istio-crds.yaml`][a]** | Creates CRDs before installing Istio.                                  |
+| **[`istio.yaml`][b]**      | Install Istio with service mesh enabled (automatic sidecar injection). |
+| [`istio-lean.yaml`][c]     | Install Istio and disable the service mesh by default.                 |
 
-[a]: https://github.com/knative/serving/releases/download/v0.2.3/istio-crds.yaml
-[b]: https://github.com/knative/serving/releases/download/v0.2.3/istio.yaml
-[c]: https://github.com/knative/serving/releases/download/v0.2.3/istio-lean.yaml
+**Install files in bold are the recommended standard install suitable for most use cases.**
```

Comment:
This info should be inline. Better yet, since we already made the point that this guide if for adv. users, maybe just remove all the bold text and simple use a "tip" up above?